fs/hostfs/hostfs.c

```diff
@@ -590,6 +593,7 @@ static int hostfs_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
   fs    = inode->i_private;
 
   DEBUGASSERT(fs != NULL);
+  UNUSED(fs);
```

Comment:
the assert look like unnecessary, let's remove it?

Reply:
I mean remove ALL `DEBUGASSERT(fs != NULL);` and `FAR struct hostfs_mountpt_s *fs;` from this file